frontend/pages/user/forgot-password.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { FormEvent, useEffect, useState } from 'react';
+import { Montserrat } from 'next/font/google';
+
+import styles from '../../styles/User.module.css';
+import { doForgotPassword } from '../api/user';
+import LoadingOverlay from '../../components/LoadingOverlay';
+
+const montserrat = Montserrat({ subsets: ['latin'] });
+
+export default function Forgot_password() {
+
+  const [isLoading, setLoading] = useState(true);
+  const [error, setError] = useState<string | null>(null);
+  const [message, setMessage] = useState<string | null>(null);
+
+  const handleForgotPassword = (e: FormEvent<HTMLFormElement>) => {
+    setLoading(true);
+    e.preventDefault();
+
+    const reqData = {
+      email: e.currentTarget.userEmail.value,
+    } 
+
+    doForgotPassword(reqData, setMessage, setError).finally(() => setLoading(false));
```

Comment:
Instead of passing in the setters from `useState`, could define a method that takes in an argument of what to set for the targeted variable. 

Example
```
const updateMessage = (message: string) => {
  setMessage(message)
}
```

However, better way to further handle this is probably to use `.then()` to check for response and use `setMessage()` to set the message based on said response.